jni/CMakeLists.txt

```diff
@@ -143,12 +147,20 @@ if (${CONFIG_FAISS} STREQUAL ON OR ${CONFIG_ALL} STREQUAL ON OR ${CONFIG_TEST} S
     endif ()
 
     # Check if patch exist, this is to skip git apply during CI build. See CI.yml with ubuntu.
-    find_path(PATCH_FILE NAMES 0001-Custom-patch-to-support-multi-vector.patch PATHS ${CMAKE_CURRENT_SOURCE_DIR}/patches/faiss NO_DEFAULT_PATH)
+    find_path(PATCH_FILE NAMES 0001-Custom-patch-to-support-multi-vector.patch 0002-Custom-patch-to-support-sqfp16-neon.patch PATHS ${CMAKE_CURRENT_SOURCE_DIR}/patches/faiss NO_DEFAULT_PATH)
```

Comment:
missing 0003-Custom-patch-to-support-AVX2-Linux-CI.patch

Reply:
That patch is only for our CI on Linux

Reply:
CI is for github. We need a way to enable AVX2 while running `gradlew build` in linux local.

Reply:
This patch is not to enable AVX2. AVX2 can be enabled by setting this optional flag `enable_SIMD` to `true`. This patch is to replace one AVX2 intrinsic on Linux only for our CI because of the CentOs that we are using in the docker image of our CI.

Reply:
You had that patch to get executed inside gradle.build before. https://github.com/opensearch-project/k-NN/pull/1421/commits/3e9bbb1213837b6f527ffe11c54c2099fe7ea61b

Why we didn't execute the patch only inside CI before then?

Reply:
There is no harm to apply this patch inside build.gradle. But, it is not required to apply everywhere on all machines using linux.

Also, before I didn't realize that we can apply it from CI. So, I instead ran that script from build.gradle